components/dsp/__test__/Support.test.tsx

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import { render, fireEvent, waitFor } from "@testing-library/react";
+import Support from "../../../components/support/Support";
+import { Provider } from "react-redux";
+import store from "../../../redux/store/store";
+import {
+  mockTraineeProfile,
+  mockTraineeProfileNoGMC
+} from "../../../mock-data/trainee-profile";
+import {
+  updatedTraineeProfileData,
+  updatedTraineeProfileStatus
+} from "../../../redux/slices/traineeProfileSlice";
+
+describe("Support", () => {
+  it("should show the tisId and GMC number plus User Agent data in the Support link href", () => {
+    jest
+      .spyOn(window.navigator, "userAgent", "get")
+      .mockReturnValue("Mocked User Agent Info");
+    jest.spyOn(window, "location", "get").mockReturnValue({
+      ...window.location,
+      href: "http://example.com"
```

Comment:
What, not evertonfc.com? :trollface:

Reply:
I'll fix this egregious error @ReubenRobertsHEE

Reply:
Be sure not to let it happen again, is all.